host-components/src/keep-manager-tester.rs

```diff
@@ -0,0 +1,172 @@
+extern crate reqwest;
+#[macro_use]
+extern crate serde_derive;
+
+use ::host_components::*;
+use reqwest::blocking::Response;
+use std::collections::HashMap;
+//TODO - better user input
+use std::io;
+
+fn main() {
+    let mut user_input = String::new();
+
+    let mut command1: HashMap<String, String> = HashMap::new();
+    command1.insert("command".to_string(), "list-all".to_string());
+    let mut command2: HashMap<String, String> = HashMap::new();
+    command2.insert("command".to_string(), "new-keep".to_string());
+    command2.insert("keep-arch".to_string(), "wasi".to_string());
+    command2.insert("auth-token".to_string(), "a3f9cb07".to_string());
+    let mut command3: HashMap<String, String> = HashMap::new();
+    command3.insert("command".to_string(), "list-keeps".to_string());
+
+    /*
+       //    let args: Vec<String> = std::env::args().collect();
+       //    let kuuid = args[1].clone();
+       let mut command4: HashMap<String, String> = HashMap::new();
+       command4.insert("command".to_string(), "start-keep".to_string());
+       command4.insert("kuuid".to_string(), kuuid);
+    */
+    println!("Welcome to the Enarx keep-manager tester.");
+    println!("We will step through a number of tests.  First ensure that you are running a");
+    println!("keep-manager on localhost port 3030 (the default).");
+    println!("");
+    println!("First test is against unimplemented backend code, and should fail!");
+    println!("Press <Enter>");
+    io::stdin()
+        .read_line(&mut user_input)
+        .expect("Failed to read line");
+
+    //NOTE - this should fail: not currently implemented
+    let builder = reqwest::blocking::Client::builder()
+        .danger_accept_invalid_certs(true)
+        .build()
+        .unwrap()
+        .post("https://localhost:3032/contracts_post/")
+        .json(&command1);
+    let res = builder.send();
+    println!("{:#?}", res);
+
+    println!("");
+    println!("Press <Enter> to create a Keep");
+
+    io::stdin()
+        .read_line(&mut user_input)
+        .expect("Failed to read line");
+
+    //construct a couple of keeps with command1
+    let res1: reqwest::blocking::Response = reqwest::blocking::Client::builder()
+        .danger_accept_invalid_certs(true)
```

Comment:
I'm trying to avoid making assumptions about what environment this tester will be running in.  And it's a tester only at this point.  Agree that this will need fixing in real life.